app/frontend/src/components/admin/AdminPage.vue

```diff
@@ -27,8 +28,18 @@ export default {
   },
   data() {
     return {
+      index: 2,
       adminDashboardUrl: this.$config.adminDashboardUrl,
     };
   },
+  computed: {
+    ...mapGetters('form', ['isRTL', 'multiLanguage']),
```

Comment:
What is `multiLanguage` used for?

Reply:
I forgot to remove it. I was trying to work on the tab issue